front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/notification/NotificationDAO.java

```diff
@@ -0,0 +1,22 @@
+package com.netflix.spinnaker.front50.model.notification;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+public interface NotificationDAO {
+
+  Collection<String> NOTIFICATION_FORMATS =
+      Arrays.asList("bearychat", "email", "googlechat", "hipchat", "pubsub", "slack", "sms");
```

Comment:
I know the original groovy didn't have it either, but it would be worth sticking a Collections.unmodifiableList around this